ETBezierEditingFreeLayout.m

```diff
@@ -10,7 +10,7 @@
 #import <EtoileUI/ETLayoutItem.h>
 #import <EtoileUI/EtoileUIProperties.h>
 #import <EtoileUI/ETSelectTool.h>
-#import <EtoileUI/NSView+Etoile.h>
+#import <EtoileUI/NSView+EtoileUI.h>
```

Comment:
I just tried to compile your code, but this import is breaking the compilation (NSView+EtoileUI.h is private). If you remove it, the code is compiles fine.